cypress/e2e/gui/mudarTema.cy.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+
+describe('Validar Cabeçalho e Mudar o Tema', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/')
+
+    });
+
+    it('Tema escuro para claro', () => {
+
+        cy.get('#theme-toggle-button')
+            .should('be.visible')
+            .click()
+
+        cy.get('h1').should('exist') // Verifica que o h1 existe dentro do cabeçalho
+            .and('be.visible') // Verifica que o h1 está visível
```

Comment:
Se um elemento está visível, ele obviamente existe, portanto, o comando anterior `.should('exist')` se torna irrelavante.

Reply:
obrigada pela dica